src/writer/delete-space.ts

```diff
@@ -21,6 +26,8 @@ export async function action(body): Promise<void> {
 
   try {
     await markSpaceAsDeleted(space);
+    await refreshProposalsCount([space]);
+    await refreshVotesCount([space]);
```

Comment:
Now that we handle them separately. we should delete the count instead of refresh?

because we have `WHERE spaces.deleted = 0` condition 🤔

Reply:
That's correct, that will come with the PR which handle the space deletion/archive. As long as the soft-delete is here, this is still needed